sunshine/config.cpp

```diff
@@ -754,6 +758,10 @@ int parse(int argc, char *argv[]) {
     }
   }
 
+  if(!fs::exists(sunshine.config_file.c_str())) {
+      sunshine.config_file = SUNSHINE_DEFAULT_DIR "/sunshine.conf";
```

Comment:
I like the idea of having a default config file in the event that the config file the user wants not yet exists.

However, later on you'll overwrite what the user wanted. I think a better method would be:

If `config_file` doesn't exist, copy contents of default config file to `config_file`: This will allow the end user to configure the path to the config file.